extensions/amp-carousel/0.1/slides.js

```diff
@@ -417,6 +421,22 @@ export class AmpSlides extends BaseCarousel {
     return this.currentIndex_ < this.slides_.length - 1;
   }
 
+  /** @override */
+  interactionNext() {
+    if (!this.nextButton_.classList.contains('amp-disabled')) {
+      this.isAutoplayRequested_ = false;
+      this.go(1, true);
+    }
```

Comment:
that should already be the case for every call to the `slides.js` `goCallback` implementation.